src/Command/StokeCommand.php

```diff
@@ -94,18 +95,31 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $metadataPath   = $input->getArgument('metadataPath');
         $caPath         = $input->getOption('certPath');
         $mailErrors     = $input->getOption('mailErrors');
+        $outputURL     = $input->getOption('outputURL');
+
 
         $this->initLogger($output, $mailErrors);
 
         $this->logger->addNotice(
             "Starting Stoke",
-            array('directory' => $directory, 'metadatPath' => $metadataPath, 'caPath' => $caPath, 'mailErrors' => $mailErrors)
+            array('directory' => $directory, 'metadatPath' => $metadataPath, 'caPath' => $caPath, 'mailErrors' => $mailErrors, 'outputURL' => $outputURL)
         );
 
         $this->verifyDestinationDirectory($directory);
+	    $this->metadataDirectory = realpath($directory) . DIRECTORY_SEPARATOR;
```

Comment:
Tab character here is not conform PSR-1: https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-1-basic-coding-standard.md